dashboard/config/environments/development.rb

```diff
@@ -73,4 +73,6 @@
   # Set "levelbuilder_mode: true" in locals.yml if you want to be able to create
   # levels or test levelbuilder functionality.
   config.levelbuilder_mode = CDO.with_default(false).levelbuilder_mode
+
+  config.experiment_cache_time_seconds = 1
```

Comment:
Do we expect any benefit from a cache with a max cache age of one second? If not, should this be set to zero?